javascript-light-bulb/main.js

```diff
@@ -1,9 +1,15 @@
 var lightBulbClicker = document.querySelector('.light-bulb-on');
 var background = document.querySelector('.light-background');
-
+var i = 0;
 function onOff(event) {
-  lightBulbClicker.className = 'light-bulb-off';
-  background.className = 'black-background';
+  i++;
+  if (i % 2 === 0) {
```

Comment:
In your code, you are using the count here only to determine if it is even or odd. There's another data type that is better to use for values that are one thing or the other thing: a boolean. You should instead make a boolean variable to keep track of if the light is currently on or not.